book/security.md

```diff
@@ -1143,8 +1145,8 @@ class JSContext:
         # ...
 ```
 
-The `allowed_request` check needs to handle both the case of no
-`Content-Security-Policy` and the case where there is one:
+The `allowed_request` check needs to handle both the case where there
+is no `Content-Security-Policy` and the case where there is one:
```

Comment:
This edit may seem low value, but LaTeX really struggled here with justification this will help it out.